src/backend/distributed/planner/multi_logical_optimizer.c

```diff
@@ -1145,12 +1163,15 @@ TransformSubqueryNode(MultiTable *subqueryNode)
 	MultiExtendedOp *masterExtendedOpNode = MasterExtendedOpNode(extendedOpNode);
 	MultiExtendedOp *workerExtendedOpNode = WorkerExtendedOpNode(extendedOpNode);
 	MultiPartition *partitionNode = CitusMakeNode(MultiPartition);
-
+	List *groupTargetEntryList = NIL;
```

Comment:
There doesn't seem to be much difference here (or not at all). Should we undo this part of the change?